go/vt/proto/automation/automation.pb.go

```diff
@@ -5,9 +5,8 @@ package automation
 
 import (
 	fmt "fmt"
-	math "math"
-
 	proto "github.com/golang/protobuf/proto"
+	math "math"
```

Comment:
This is a conflict between grpc code gen and goimports. If you re-run goimports, all these files will revert to unchanged. Or, you can just manually revert these yourself.